pkg/apis/metering/v1/metering_config.go

```diff
@@ -453,9 +452,6 @@ type PrestoConnectorHiveConfig struct {
 	MetastoreTimeout       string     `json:"metastoreTimeout,omitempty"`
 	TLS                    *TLSConfig `json:"tls,omitempty"`
 }
-type PrestoConnectorPrometheusConfig struct {
-	Enabled *bool `json:"enabled,omitempty"`
-}
```

Comment:
Why was this deleted?

Reply:
accident, putting it back. I got confused that we wanted the default case, but if default then no Prom connector.